internal/provider/provider.go

```diff
@@ -92,6 +92,7 @@ func New() *schema.Provider {
 			"boundary_host_set_plugin":              resourceHostSetPlugin(),
 			"boundary_role":                         resourceRole(),
 			"boundary_scope":                        resourceScope(),
+			"boundary_self_managed_worker":          resourceSelfManagedWorker(),
```

Comment:
Hmm, I wonder if the name should be `self_managed_worker` instead of `worker`, We generally try and align the CLI/API with terraform provider and on the API/CLI we just use `worker`

Reply:
Happy to change this.